interfaces/builtin/docker_support.go

```diff
@@ -157,7 +172,8 @@ ptrace (read, trace) peer=docker-default,
 # needed by runc for mitigation of CVE-2019-5736
 # For details see https://bugs.launchpad.net/apparmor/+bug/1820344
 / ix,
-/bin/runc rix,
+/bin/runc ixr,
+/pause ixr,
```

Comment:
`/pause` is an interesting path. If you know how it happens to work I think a comment would be useful here.